var/spack/repos/builtin/packages/faodel/package.py

```diff
@@ -37,7 +37,7 @@ class Faodel(CMakePackage):
     depends_on('hdf5+mpi', when='+hdf5+mpi')
     depends_on('hdf5~mpi', when='+hdf5~mpi')
     depends_on('libfabric@1.5.3:', when='network=libfabric')
-    depends_on('googletest@1.7.0:', type='build')
```

Comment:
The failure might be legitimate, since it's `faodel` failing in the `e4s` pipelines: https://gitlab.spack.io/spack/spack/-/jobs/478592

Reply:
Yeah looks like the package that builds against Google test legitimately links against it even when not building unit tests... 🙄

Reply:
I was wrong, it looks like it just defaulted to `BUILD_TESTS=ON` and so required googletest by default. Tested it locally without the dependency and it works, fingers crossed for the gitlab ci build (and hey this might mean one fewer package to have to build with the E4S ci!)